rootfs/etc/profile.d/defaults.sh

```diff
@@ -31,7 +31,6 @@ fi
 #
 # Helm
 #
-export HELM_HOME=${REMOTE_STATE}/helm/
```

Comment:
The problem is we install helm plugins during build but have a `HELM_HOME` that is expected to be mounted.

I think it should be instead:
 ```
ENV HELM_HOME /var/lib/helm
```

We want a consistent location for these. By not setting it, it will use `/root/.helm`.

Reply:
hm... 
We define `HELM_HOME` on docker build 
https://github.com/cloudposse/geodesic/blob/fix-helm-home/Dockerfile#L75
When `/etc/profile.d/defaults.sh` runs `HELM_HOME` is already defined and point to `/var/lib/helm`

I do not see any reason why we should override it in `/etc/profile.d/defaults.sh`

Reply:
Why we need `HELM_HOME` to be mounted?